roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
We really just don't want to have more images than we need right now.  I'd say it's on the table for the future but until we start shipping something that people find useful we want to multiple options for them.